src/cli/plugin.rs

```diff
@@ -0,0 +1,116 @@
+use crate::serve_session::ServeSession;
+
+use memofs::InMemoryFs;
+use memofs::{Vfs, VfsSnapshot};
+use roblox_install::RobloxStudio;
+use snafu::{ResultExt, Snafu};
+use std::{
+    fs::{self, File},
+    io::{self, BufWriter},
+};
+use structopt::StructOpt;
+
+static PLUGIN_FILE_NAME: &str = concat!("rojo-", env!("CARGO_PKG_VERSION"), ".rbxmx");
+
+#[derive(Debug, StructOpt)]
+enum PluginSubcommand {
+    /// Install the plugin in Roblox plugins folder. If the plugin is already installed, installing
+    /// it again will overwrite the current plugin file.
+    Install,
+    /// Removes the plugin if installed.
+    Uninstall,
+}
+
+/// Install rojo's plugin.
+#[derive(Debug, StructOpt)]
+pub struct PluginCommand {
+    #[structopt(subcommand)]
+    subcommand: PluginSubcommand,
+}
```

Comment:
Can we move all of the `StructOpt` code into `src/cli/mod.rs`? I think it's nice to keep all of it contained in one spot.